### ISSUE TITLE
This is more of a feature request for some clarifications in the ReadMe

### ISSUE THREAD

**Issue by ChantalJuntao** [HolyLab/HolyLabRegistry#94]
I'm only struggling with this because I'm a complete beginner to package development. But since the ReadMe contains a section on using Package Templates to generate new packages, and many lab members who have never generated a new package before may also, like me, have never tracked a new package using git before, can we also include a section on tracking new packages with git and pushing them up to github? As I've never tracked a new package before, I don't know if I could make any decisions that could make the package incompatible with the registry.

**Comment by kdw503** [comment]
Are you asking how to a push newly generated package to github for the first time?
If so, I'll add a related section to our README. This is good idea.

**Comment by ChantalJuntao** [comment]
Yes, that would be very helpful. There aren't many guides specific to Julia out there.

**Comment by kdw503** [comment]
If you make repository in here(our repository website) with same name as your local one but not including README file and gitignore file, then you can push your local repository to here without any conflict.
